extensions/WEBGL_shader_pixel_local_storage/extension.xml

```
@@ -115,53 +124,44 @@ interface WEBGL_shader_pixel_local_storage {
       <param name="plane" type="GLint"/>
       <param name="value" type="Float32List"/>
-      <param name="src_offset" type="GLuint" default="0"/>
+      <param name="srcOffset" type="GLuint" default="0"/>
       Sets the floating point clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>src_offset + 4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
+      <p/>
+      If <code>value</code> has less than <code>srcOffset + 4</code> elements, generates an
+      <code>INVALID_VALUE</code> error.
     </function>
   </newfun>
 
   <newfun>
     <function name="framebufferPixelLocalClearValueivWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
       <param name="value" type="Int32List"/>
-      <param name="src_offset" type="GLuint" default="0"/>
+      <param name="srcOffset" type="GLuint" default="0"/>
       Sets the integer clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>src_offset + 4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
+      <p/>
+      If <code>value</code> has less than <code>srcOffset + 4</code> elements, generates an
     </function>
   </newfun>
 
   <newfun>
     <function name="framebufferPixelLocalClearValueuivWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
       <param name="value" type="Uint32List"/>
-      <param name="src_offset" type="GLuint" default="0"/>
-      Sets the unsigned integer clear value for the given plane.
-      <div class="note">
-        If <code>value</code> has less than <code>src_offset + 4</code> elements, generates an
-        <code>INVALID_OPERATION</code> error.
-      </div>
+      <param name="srcOffset" type="GLuint" default="0"/>
+      <p/>
+      If <code>value</code> has less than <code>srcOffset + 4</code> elements, generates an
```

It looks like the documentation and INVALID_OPERATION error were lost from the docs here.

---

Done